### ISSUE TITLE
VPTeamMembers requires avatar to get displayed correctly

### ISSUE THREAD

**Issue by MarkusKeck** [vuejs/vitepress#1073]
### Describe the bug

If you use the VPTeamMembers component provided by vitepress you are forced to use an avatar with it.
Otherwise the browser displays a missing image.
If you dont have an image of a person, you cannot use this component or have to design an own "missing" avatar icon.

### Reproduction

Use the VPTeamMembers component in any page and have an empty avatar attribute.

```yml
 {
    avatar: '',
    name: 'Evan You',
    title: 'Creator',
    links: [
      { icon: 'github', link: 'https://github.com/yyx990803' },
      { icon: 'twitter', link: 'https://twitter.com/youyuxi' }
    ]
  }
```

### Expected behavior

Component should not rely on that avatar image.

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (32) x64 AMD Ryzen 9 3950X 16-Core Processor
    Memory: 106.32 GB / 127.91 GB
  Binaries:
    Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.7.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (103.0.1264.71)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    vitepress: ^1.0.0-alpha.4 => 1.0.0-alpha.4
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by kiaking** [comment]
This is intended. The card is exclusively designed with an avatar image.

I'll mark this as an enhancement. I see there's a demand for text only team page component, but in that case, I would like to try few designes. Mainly, if there's no image, I think aligning text left might be better looking 🤔

**Comment by MarkusKeck** [comment]
Thanks for the fast reply. I think the general problem I have with this at the moment is, that this is the only available component for vitepress. There is not even a simple text component which you can use in the home layout, so I tried using the Team Component for simple text display. Sure I could just build my own with vue, and I know that vitepress doesnt want to have the same level of complexity as vuepress, but are there any plans to include some more really basic components to vitepress which you could use inside the home layout?

**Comment by brc-dd** [comment]
Let's fix the main issue at #800 (content inside home layout). Also, IG there are other components (like VPFeatures) exported too now. And, regarding the avatar thing, I don't see a particular demand for that. Gonna close this as not planned (wont fix + stale).
